core/domain/exp_services_test.py

```diff
@@ -1059,21 +1535,21 @@ def test_get_image_filenames_from_exploration(self):
                     '<oppia-noninteractive-image filepath-with-value='
```

Comment:
Why the file-with-value still exist here?

Reply:
Since it is the unit test of the function ```get_image_filenames_from_exploration ```. so for testing it the ``` filepath ``` exist here. But I found that these function is not using anywhere. So can I remove this function?